tests/cases/fourslash/jsxElementMissingOpeningTagNoCrash.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference path="fourslash.ts" />
+
+//@Filename: file.tsx
+//// let x = <></Fo/*$*/o>;
+
+verify.quickInfoAt("$", "let Foo: any")

```

Comment:
This fails right now as it says "any"; I think this test case was wrong originally given I didn't actually declare `Foo`.